src/frontend/src/pages/FinancePage/ReimbursementRequestForm/ReimbursementProductTable.tsx

```diff
@@ -97,41 +98,51 @@ const ReimbursementProductTable: React.FC<ReimbursementProductTableProps> = ({
-                                }}
-                                onBlur={(e) => onCostBlurHandler(parseFloat(e.target.value), product.index)}
-                                sx={{ width: '50%' }}
-                                error={!!errors.reimbursementProducts?.[product.index]?.cost}
-                              />
-                            )}
-                          />
+                          <div style={{ display: 'flex', flexDirection: 'column', width: '50%', marginRight: '4px' }}>
+                            <Controller
+                              name={`reimbursementProducts.${product.index}.name`}
+                              control={control}
+                              render={({ field }) => (
+                                <TextField
+                                  {...field}
+                                  label={'Description'}
+                                  autoComplete="off"
+                                  size={'small'}
+                                  variant={'outlined'}
+                                  sx={{ width: '100%' }}
+                                  error={!!errors.reimbursementProducts?.[product.index]?.name}
+                                />
+                              )}
+                            />
+                            <FormHelperText error>
+                              {errors.reimbursementProducts?.[product.index]?.name?.message}
+                            </FormHelperText>
+                          </div>
+                          <div style={{ display: 'flex', flexDirection: 'column', width: '50%' }}>
+                            <Controller
+                              name={`reimbursementProducts.${product.index}.cost`}
+                              control={control}
+                              render={({ field }) => (
+                                <TextField
+                                  {...field}
+                                  label={'Cost'}
+                                  size={'small'}
+                                  variant={'outlined'}
+                                  type="number"
+                                  autoComplete="off"
+                                  InputProps={{
+                                    startAdornment: <InputAdornment position="start">$</InputAdornment>
+                                  }}
+                                  onBlur={(e) => onCostBlurHandler(parseFloat(e.target.value), product.index)}
+                                  sx={{ width: '100%' }}
+                                  error={!!errors.reimbursementProducts?.[product.index]?.cost}
+                                />
+                              )}
+                            />
+                            <FormHelperText error>
+                              {errors.reimbursementProducts?.[product.index]?.cost?.message}
```

Comment:
I realized we get the following long non-user-friendly error msg under the cost box when we delete the whole value and try to rewrite the cost value. Can we fix this, so the cost box error msg stays the same as `Amount must be greater than 0`?

<img width="960" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/91857946/17a36d52-9a58-44d9-a02a-49e4eded7ed4">